gui/src/settings.cpp

```diff
@@ -106,6 +130,25 @@ unsigned int Settings::GetAudioBufferSize() const
 	return v ? v : GetAudioBufferSizeDefault();
 }
 
+QAudioDeviceInfo Settings::GetAudioOutDevice() const
+{
+	QAudioDeviceInfo outDev;
+
+	QString s = settings.value("settings/audio_out_device", "default").toString();
+	for (QAudioDeviceInfo di : QAudioDeviceInfo::availableDevices(QAudio::AudioOutput))
+	{
+		if(di.deviceName() == s)
+			outDev = di;
```

Comment:
Remove the outDev variable and early-return here:
```suggestion
			return di;
```